backend/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+authors = ["Mark-Simulacrum <mark.simulacrum@gmail.com>"]
```

Comment:
`["Mark-Simulacrum <mark.simulacrum@gmail.com>", "Nicholas Cameron <ncameron@mozilla.com>", "The rustc-perf contributors"]`